lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +25,67 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class MobileAppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    latest_version_android = TextField(
+        blank=False,
+        help_text="Latest available version for android app in X.X.X format"
+    )
+    latest_version_ios = TextField(
+        blank=False,
+        help_text="Latest available version for IOS app in X.X.X format"
+    )
+    min_supported_version_android = TextField(
+        blank=False,
+        help_text="Min supported version for android app in X.X.X format"
+    )
+    min_supported_version_ios = TextField(
+        blank=False,
+        help_text="Min supported version for IOS app in X.X.X format"
+    )
+    next_supported_version_android = TextField(
+        blank=False,
+        help_text="Next supported version for android app in X.X.X format that a user must upgrade to before deadline"
+    )
+    next_supported_version_ios = TextField(
+        blank=False,
+        help_text="Next supported version for IOS app in X.X.X format that a user must upgrade to before deadline"
+    )
+    next_update_required_android = DateTimeField(
```

Comment:
On second thoughts `next_update_required` sounds like a boolean. Lets make this `update_required_date`.